components/script/dom/document.rs

```diff
@@ -549,7 +549,6 @@ impl Document {
     pub fn send_title_to_compositor(&self) {
         let window = self.window();
         // FIXME(https://github.com/rust-lang/rust/issues/23338)
-        let window = window.r();
```

Comment:
let compositor = self.window().compositor() and remove the above line and FIXME?